.github/workflows/reusable-release-package.yml

```
@@ -0,0 +1,146 @@
+name: Release package
+
+on:
+  workflow_call:
+    inputs:
+      # The GitHub App that has been given permission to push to repo.
+      app-id:
+        type: "string"
+        required: true
```

```suggestion
        required: true
      publish:
        description: "Whether to publish to PyPI or not."
        type: "boolean"
        required: false
        default: false
```

We can include a condition here to either publish or not.